### ISSUE TITLE
Repository is empty

### ISSUE THREAD

**Issue by augusto-herrmann** [DataShades/datashades-info#2]
Why is this repository empty? Is this not the source code to the site deployed at [datashades.info](https://datashades.info), with information on CKAN deployments around the world? Which also had a [blog post](https://blog.okfn.org/2019/09/23/introducing-datashades-info-a-ckan-community-service/) at Open Knowledge in 2019?

**Comment by AlekseyPavlyuk** [comment]
> Why is this repository empty? Is this not the source code to the site deployed at [datashades.info](https://datashades.info), with information on CKAN deployments around the world? Which also had a [blog post](https://blog.okfn.org/2019/09/23/introducing-datashades-info-a-ckan-community-service/) at Open Knowledge in 2019?

Hello Augusto
For now, we store the code in our (LinkDigital) private repository. We plan to refactor and publish it this year.

**Comment by augusto-herrmann** [comment]
Oh, I see. Good thing there is at least this issue tracker here, so I can report problems with the site.

**Comment by AlekseyPavlyuk** [comment]
> Oh, I see. Good thing there is at least this issue tracker here, so I can report problems with the site.

Yes, feel free to report here any issues you will find in the future with that project and we will fix everything. We still support the project. And thank you for the bug report.
